utils.py

```diff
@@ -1940,6 +1955,53 @@ def minimal_realisation(a, b, c):
 
     return Aco, Bco, Cco
 
+def num_denom (A, symbolic_expr = False):
+
+    sym_den = 0
+    sym_num = 0
+    s = sympy.Symbol('s')
+
+    if type(A) == mimotf:
+        denom   = 1
+        num     = 1
+        
+        for j in range(A.matrix.shape[1]):
+            denom = list(numpy.poly1d(denom) * numpy.poly1d(A.matrix[0,j].denominator.coeffs))
+            num   = list(numpy.poly1d(num)   * numpy.poly1d(A.matrix[0,j].numerator.coeffs))
+            if symbolic_expr == True:
+                for n in range(len(denom)):
+                    sym_den = (sym_den + denom[len(denom) - n- 1] * s**n).simplify()
```

Comment:
Negative indexes go from the back in Python, so `denom[len(denom)-1]` is the same as `denom[-1]`